code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -55,6 +55,7 @@
 	desc = "An energy-based laser gun that draws power from the cyborg's internal energy cell directly. So this is what freedom looks like?"
 	origin_tech = null
 	use_cyborg_cell = 1
+	fire_delay = 50 ///Should be around the same as a swarmer

```

Comment:
>50 deciseconds

that's far far FAR more of a nerf than halved.
normal firing rate is like 4 deciseconds wtf anonmare.

Reply:
i don't see anything wrong here kev 😏